cmake/Modules/FindAzureSDK_EP.cmake

```diff
@@ -94,6 +94,10 @@ if (NOT AZURESDK_FOUND)
     endif()
 
     if (WIN32)
+      if(MSVC)
+        set(CXXFLAGS_DEF " -I${TILEDB_EP_INSTALL_PREFIX}/include /Dazure_storage_lite_EXPORTS /DCURL_STATICLIB=1 ${CMAKE_CXX_FLAGS}")
```

Comment:
In theory the only thing that should make a difference here is `/DCURL_STATICLIB=1`.